include/WCSimDetectorConstruction.hh

```diff
@@ -60,6 +60,9 @@ public:
 
   G4VPhysicalVolume* Construct();
 
+  // Temporary, where should I place this line?
+  G4bool placeBGOGeometry;
```

Comment:
Remember to initialize it in WCSimDetectorConstruction.cc

Reply:
👍